address-space-controller/src/main/java/io/enmasse/controller/common/NoneAuthenticationServiceResolver.java

```diff
@@ -44,7 +45,8 @@ public int getPort(AuthenticationService authService) {
 
     @Override
     public Optional<String> getSaslInitHost(AddressSpace addressSpace, AuthenticationService authService) {
```

Comment:
I guess we're saying that this field is really required now, does it make sense to change it to return a String instead of the optional?

Reply:
Yes, that probably does make sense.